backend/tests/Feature/Quality/NonConformanceReportTest.php

```
@@ -0,0 +1,113 @@
+        IssueAction::factory()->overdue()->create(['issue_id' => $issue->id]);
+        // Future due date — not overdue.
+        IssueAction::factory()->create(['issue_id' => $issue->id, 'due_date' => now()->addWeek()->toDateString()]);
+        // Past due but already verified — not outstanding.
+        IssueAction::factory()->verified()->create(['issue_id' => $issue->id, 'due_date' => now()->subWeek()->toDateString()]);
+
+        $this->assertSame(2, app(NonConformanceReportService::class)->overdueActionsCount());
+    }
+
+    public function test_api_pareto_endpoint_returns_data(): void
+    {
+        $type = IssueType::factory()->create(['name' => 'Crack']);
```

PHPStan (2.2.2)

[error] Call to an undefined static method App\Models\IssueType::factory().


(staticMethod.notFound)